### DISCUSSION TITLE
A black tetromino

### DISCUSSION THREAD

**Discussion by SundayLee404** [MrStahlfelge/lightblocks#43] [Q&A]
Why it will happened?
That Black I piece
![image](https://user-images.githubusercontent.com/82038537/125739622-1f76dfcd-739f-4b59-9f6a-ee816248ec9b.png)

**Comment by MrStahlfelge** [comment]
This can sometimes happen when resuming a saved game. If it happens, consider yourself the lucky - something good will happen. ;)

**Reply by SundayLee404** [reply]
but I didnt pause and resume(it's a fully new game)
this happened when I press hold(expect first hold and hold not enable) in a pretty short time

**Reply by MrStahlfelge** [reply]
Ah, thanks, I will check that!

**Reply by MrStahlfelge** [reply]
I couldn't confirm the behaviour, with which input method (controller, keyboard, gestures, on screen buttons) does this happen for you?

**Reply by SundayLee404** [reply]
umm
keyboard, screen buttons and gestures(or ALL maybe）

**Reply by SundayLee404** [reply]
or I record a video maybe(use screen buttons)

**Reply by MrStahlfelge** [reply]
yes, please do, I can't reproduce.

**Reply by SundayLee404** [reply]
like this(pretty laggy)
https://user-images.githubusercontent.com/82038537/127686523-c91199fe-8bb5-466a-9b29-74cca477cb7c.mp4

**Reply by SundayLee404** [reply]
I thought I'll record it again on my mobile

**Reply by MrStahlfelge** [reply]
thanks, I can reproduce when trying on your speed

**Reply by SundayLee404** [reply]
But my mobile is confiscated by my dad now... then maybe need to wait until I fully done my works that my mom want me to do(At least 2 weeks I thought)
